classes/PaymentModule.php

```diff
@@ -457,7 +457,7 @@ public function validateOrder(
                     'id_product' => $product['id_product'],
                     'id_product_attribute' => $product['id_product_attribute'],
                     'reference' => $product['reference'],
-                    'name' => $product['name'] . (isset($product['attributes']) ? ' - ' . $product['attributes'] : ''),
+                    'name' => $product['name'] . (isset($product['attributes']) && $product['attributes'] != '' ? ' - ' . $product['attributes'] : ''),
```

Comment:
```suggestion
                    'name' => (!empty($product['attributes']) ? ' - ' . $product['attributes'] : ''),
```

maybe something simpler?